Expensify/App
# [Production QA] Update app description title and preview image

cc @davidcardoza 

### Details
Update the `title` property of the app's meta tags to `New Expensify`.

### Fixed Issues
$ https://github.com/Expensify/App/issues/5737#issuecomment-939192158
Follow-up of https://github.com/Expensify/App/pull/5740 that got merged too soon.

### Tests
1. Launch ngrok so you can have a public-facing URL for NewDot
2. Copy the URL in an app that does "previews", e.g. Whatsapp, Twitter, FB, etc.
3. Make sure the title of the app in the preview is `New Expensify`
3. Make sure the logo in the preview is the "E" version with a blue background - _not_ the "$" with a white background

### QA Steps
1. In WhatsApp or Twitter or FB, post a link to https://new.expensify.com
2. Make sure the title of the app in the preview is `New Expensify`, like on the screenshot
3. Make sure the logo in the preview is the "E" version with a blue background - _not_ the "$" with a white background
![image](https://user-images.githubusercontent.com/2229301/136828216-42d32100-dbae-4c31-8ef8-5be5b92237a6.png)
